app/presenters/reservation_user_action_presenter.rb

```diff
@@ -39,6 +39,9 @@ def user_actions
   end
 
   def kiosk_user_actions
+    # no actions allowed for Offline and Admin Reservations
+    return [] unless order_detail.present?
```

Comment:
Can we make OfflineReservation and AdminReservation return falsey to the below booleans, making this unnecessary?